content/docs/dvclive/live/log_artifact.md

```diff
@@ -36,21 +37,27 @@ with Live() as live:
 
 Uses `dvc add` to track `path` with DVC, generating a `{path}.dvc` file.
 
-When combined with `save_dvc_exp=True`, it will ensure that `{path}.dvc` is
-included in the experiment.
+When combined with [`save_dvc_exp=True`](/doc/dvclive#initialize-dvclive), it
+will ensure that `{path}.dvc` is included in the experiment.
```

Comment:
Very minor, but WDYT about combining these paragraphs? It creates a 2 paragraph description that matches what the method does - 1) tracks with dvc and 2) adds artifact to dvc.yaml.

Reply:
yep, not related to this PR, but I can do it